lib/matplotlib/cbook.py

```diff
@@ -2445,6 +2445,14 @@ def safe_first_element(obj):
     return next(iter(obj))
 
 
+def asSequence(data):
```

Comment:
I was thinking about exactly that, but didn't suggest it because at least at present that's not at all what the function is doing.  In addition, I suspect this needs to happily pass scalars.